indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -0,0 +1,297 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import timeTableURL from 'indico-url:event_registration.api_event_timetable';
+
+import _ from 'lodash';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+import {useSelector} from 'react-redux';
+import {Checkbox, Form, Accordion, AccordionTitle, AccordionContent, Icon} from 'semantic-ui-react';
+
+import {FinalRadio, FinalField, FinalInput, validators as v} from 'indico/react/forms';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {Translate, Param, Plural, PluralTranslate, Singular} from 'indico/react/i18n';
+
+import '../../../styles/regform.module.scss';
+import {getStaticData} from '../selectors';
```

Comment:
move this one up a line and then let eslint fix the import ordering

the CSS import (which is considered an import w/ side effects) prevents auto reordering by eslint. so the CSS imports should always come last

Reply:
That's what I did and eslint moved it to the bottom of the imports (it was previously on top side)

Sadly my linters are not working very well in indico core, but if you tell me where to place them I can do it with no problem

Reply:
```
import timeTableURL from 'indico-url:event_registration.api_event_timetable';

import _ from 'lodash';
import moment from 'moment';
import PropTypes from 'prop-types';
import React, {useEffect, useState} from 'react';
import {useSelector} from 'react-redux';
import {Checkbox, Form, Accordion, AccordionTitle, AccordionContent, Icon} from 'semantic-ui-react';

import {FinalRadio, FinalField, FinalInput, validators as v} from 'indico/react/forms';
import {useIndicoAxios} from 'indico/react/hooks';
import {Translate, Param, Plural, PluralTranslate, Singular} from 'indico/react/i18n';

import {getStaticData} from '../selectors';

import '../../../styles/regform.module.scss';
```